### ISSUE TITLE
Did I mess up a device driver when running the script?

### ISSUE THREAD

**Issue by GrailNorth** [ChrisTitusTech/winutil#80]
After doing a fresh install of Win 10 Pro, I ran the winutil script (the current version as of the day I did my build, June 22) with recommended settings and then ran O&OShutup, also using the recommended settings.

Now I've discovered that I can't plug in my iPad for syncing with the iTunes Win desktop app: it says that it is waiting for Windows to install an updated driver. When I check Device Manager, it is showing a driver for the iPad under "Portable Devices", but it still isn't recognized by iTunes. I've uninstalled and reinstalled iTunes, as well as the driver, many reboots, etc., but still no luck.

Win and the iTunes app are up to date.

Which part of Windows which I might have nuked in the debloating script that could be needed for this?  I'm investigating the debloat script since I did nothing else to give rise to this issue but it could certainly be something else.  Thanks

**Comment by ChrisTitusTech** [comment]
probably the windows drivers portion. 
type sysdm.cpl in run and go to drive installation method, set that to default and you should be good to go.

**Comment by GrailNorth** [comment]
Update:  Working this problem further, it’s turned into a giant headache and may not have anything to do with settings changes and debloating from running winutil.  It’s the widespread issue with “some settings are managed by your system administrator” that I can’t seem to turn off, even though I’m using the admin account. 

I went through gpedit.msc and couldn’t find any settings that weren’t “not configured”. 
I tried the gpedit.msc recommended tricks in other help forums, of 

(1) enabling and then disabling “Do not include drivers Windows with windows updates” (in Administrative Templates > Windows Components > Windows Updates, 

(2) turning telemetry on in gpedit.msc, then applying it, running gpupdate /force to update the settings and then undoing it. 

Didn't work after reboot.

So far I haven’t figured out how to take back control of the settings that are blocked on my own PC. I feel confident that if I could ever turn off “Do not include drivers Windows with windows updates” my original problem of the iPad device driver would be resolved.

I’m using Bitdefender (paid) and some forums say that if it is allowed to use a “work” profile it prevent s changes. However, I have Bitdefender profiles turned off and I still get the issue if I turn Bitdefender off.

It’s weird, I have a separate laptop with Win 10 Pro and no settings are blocked for the admin account.
